### DISCUSSION TITLE
consult-line cycle to end/beginning of the list

### DISCUSSION THREAD

**Discussion by SudoerReodus** [minad/consult#857] [Q&A] [answered]
Not sure if this is already possible to achieve by changing the configuration, but it would be nice if consult-line cycled/jumped to the end of the list when you are the top of the list and further press M-p, and likewise if it cycled/jump to the beginning of the list when you are at the end and further press M-n

Thanks

**Comment by minad** [comment, marked as answer]
If you use Vertico, you can set `vertico-cycle` to t.

**Reply by SudoerReodus** [reply]
Thanks
And damn you are quick :)
